src/build123d/topology.py

```diff
@@ -2941,7 +2944,7 @@ def to_arcs(self, tolerance: float = 1e-3) -> Face:
     def _repr_javascript_(self):
         """Jupyter 3D representation support"""
 
-        from .jupyter_tools import display
+        from .jupyter_tools import display # pylint: disable=import-outside-toplevel
```

Comment:
I'd like this one to remain.  I expected this import would work as a normal import at the top of the module but a quick test failed and I didn't have time to investigate why.

Reply:
Done.